project/src/main/java/com/google/autograder/servlets/ListAssignmentsServlet.java

```diff
@@ -34,13 +34,12 @@
 
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        if (Services.USER_SERVICE.isUserLoggedIn()) {
+        if (Database.getUserService().isUserLoggedIn()) {
```

Comment:
Similar to above, this would be better as Database.isUserLoggedIn, or even better, in a separate class (`UserService.isUserLoggedIn`)